README.md

```diff
@@ -24,51 +24,20 @@ If you prefer to use Git, you can use any Git client with TFS or Team Services a
-### Install Java 6
-1. We use JavaSE-1.6 as the minimal supported Java execution environment.
-1. Download and install the JDK for [JavaSE-1.6](http://www.oracle.com/technetwork/java/javase/downloads/java-archive-downloads-javase6-419409.html).
-1. Set the JAVA_HOME environment variable to point to the install, e.g.
- * (Windows) `SET JAVA_HOME=C:\dev\java\jdk1.6.0_45`
- * (Linux) `JAVA_HOME=~/dev/java/jdk1.6.0_45`
- * (Mac) `JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_60.jdk/Contents/Home`
-1. Add JAVA_HOME bin directory to the path
- * (Windows) `SET PATH=%JAVA_HOME%\bin;%PATH%`
- * (Linux) `PATH=$JAVA_HOME/bin:$PATH`
- * (Mac) `PATH=$JAVA_HOME/bin:$PATH`
-    
-### Install Ant 
-1. If you do not already have it, download and install Apache Ant(TM) version 1.9.6 from [Ant Binary Distributions](http://ant.apache.org/bindownload.cgi).
-1. Add the full path of the Ant `bin` directory to the `PATH` system environment variable. You can find more Ant installation details [here](http://ant.apache.org/manual/install.html#installing).
- * (Windows) `SET PATH=C:\dev\apache-ant-1.9.6\bin;%PATH%`
- * (Linux) `PATH=~/dev/apache-ant-1.9.6/bin:$PATH`
- * (Mac) `PATH=~/dev/apache-ant-1.9.6/bin:$PATH`
-    
-### Install the Eclipse Target Environment
-We use Eclipse 3.5.2 as the minimum supported Eclipse version.
-1. Download and install Eclipse Classic from [Eclipse 3.5.2](http://www.eclipse.org/downloads/packages/release/galileo/sr2). On Windows, you may want to use a third party ZIP tool to unzip the Eclipse archive.
-1. Install the [EGit 2.1.0](http://archive.eclipse.org/egit/updates-2.1) plug-in into that Eclipse instance.
+## How do I build the source code?
 
-### Clone the Repository
-Use the Git tool of your choice to clone the repository into a local path.
-For example, you could use git.exe from a Windows console window:
-```
-mkdir c:\repos
-pushd c:\repos
-git clone https://github.com/Microsoft/team-explorer-everywhere
-```
+Before you can build, please follow the [Getting Started Guide](GettingStarted.md) to install & configure the tools you'll need.
 
-### Build
-Note: The Eclipse target installation location is needed as a parameter for the Ant build variable `dir.machine.build-runtime`. For the samples below, we will assume that the target Eclipse version was installed into '\dev\eclipseTargets\352'.
-1. From a terminal/console window, change to the `build` subfolder of the root folder of the team-explorer-everywhere repository
+Note: The Eclipse target installation location is needed as a parameter for the Ant build variable `dir.machine.build-runtime`. For the samples below, we will assume that the target Eclipse version was installed into the `dev/eclipseTargets/352` sub-folder under your `HOME` folder.
+1. From a terminal/console window, change to the `build` sub-folder of the root folder of the **team-explorer-everywhere** repository.
 1. Run ant -Ddir.machine.build-runtime=`<pathToEclipseTarget>`, for example, 

```

Comment:
Why `pathTo**Target**`? Eclipse 3.5.2 we use in the build is not a target, it's a building tool.